ChangeLog

```diff
@@ -46,6 +46,10 @@ Change log for the astroid package (used to be astng)
 
 	Close PyCQA/pylint#1884
 
+  * Inference now understands the 'isinstance' builtin
+
+	 Close #98
```

Comment:
Tab-based indentation here?

Reply:
My bad, looks like vim has a 'changelog' filetype that I needed to override